src/node/server/serverPluginCss.ts

```diff
@@ -98,6 +98,20 @@ export const cssPlugin: ServerPlugin = ({ root, app, watcher, resolver }) => {
       }
       // no boundaries
       normalCssUpdate(publicPath)
+    } else if (cssImportMap.has(filePath)) {
+      const boundaries = getCssImportBoundaries(filePath)
```

Comment:
Looks like this is largely redundant with the logic above, let's extract this into a function